app/assets/stylesheets/govuk-component/govspeak/_call-to-action.scss

```diff
@@ -0,0 +1,25 @@
+// Govspeak call to action
+// http://govuk-component-guide.herokuapp.com/components/govspeak/fixtures/call_to_action
+//
+// Support:
+//
+// - alphagov/whitehall: ✔︎
+// - alphagov/govspeak: ✔︎
+
+.govuk-govspeak {
+  .call-to-action {
+    margin: 2em 0;
```

Comment:
I _guess_ not as that should be the default… this is a bit more defensive I guess. And I think I just prefer it to setting `margin-top`, `margin-bottom` and leaving the rest to chance?